tslearn/docs/examples/plot_barycenters.py

```diff
@@ -4,50 +4,84 @@
 ===========
 
 This example shows three methods to compute barycenters of time series.
+For an overview over the available methods see the :mod:`tslearn.barycenters`
+module.
+
+*tslearn* provides three methods for calculating barycenters for a given
+set of time series:
+
+* *Euclidean barycenter* is simply the arithmetic mean for
+  each individual point in time, minimizing the summed euclidean distance
+  for each of them. As can be seen below, it is very different from the
+  DTW-based methods and may often be inappropriate. However, it is the
+  fastest of the methods shown.
+* *DTW Barycenter Averaging (DBA)* is an iteratively refined barycenter,
```

Comment:
Maybe here and for the next: add references for DBA and softDTW.

Reply:
Done. I also added the reference for Soft-DTW to `tslearn.barycenters.softdtw_barycenter()`. It was missing there.